certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -208,14 +226,19 @@ def _launch_process(self, command, cwd=os.getcwd(), env=None, force_stderr=False
 
 
 def main():
-    args = sys.argv[1:]
-    server_type = args[0] if args else 'pebble'
-    possible_values = ('pebble', 'boulder-v1', 'boulder-v2')
-    if server_type not in possible_values:
-        raise ValueError('Invalid server value {0}, should be one of {1}'
-                         .format(server_type, possible_values))
-
-    acme_server = ACMEServer(server_type, [], http_proxy=False, stdout=True)
+    parser = argparse.ArgumentParser(
+        description='CLI tool to start a local instance of Pebble or Boulder CA server.')
+    parser.add_argument('--server-type', '-s',
+                        choices=['pebble', 'boulder-v1', 'boulder-v2'], default='pebble',
+                        help='type of CA server to start: can be Pebble or Boulder '
+                             '(in ACMEv1 or ACMEv2 mode), Pebble is used if not set.')
+    parser.add_argument('--dns-server', '-d',
+                        help='(Pebble specific) specify the DNS server as `IP:PORT` to use by '
```

Comment:
This isn't Pebble specific anymore is it?
```suggestion
                        help='specify the DNS server as `IP:PORT` to use by '
```